qa/tasks/tests/test_parity_consistency.py

```diff
@@ -0,0 +1,242 @@
+"""
+Unit tests for ec_parity_consistency.py
+
+Tests can be run by invoking pytest from the qa directory:
+
+cd /ceph/qa/
+pytest tasks/tests/test_parity_consistency.py
```

Comment:
I tested out these steps and thought we could make it even more accessible by adding steps for installing dependencies. Here's the steps I took for a venv:
```suggestion
python3 -m venv venv
source venv/bin/activate
pip install -r tasks/tests/requirements.txt
pytest tasks/tests/test_parity_consistency.py
deactivate (when done)
```

And here's a basic `requirements.txt` file I came up with based on running all the python test files under this dir:
```
boto
gevent
httplib2
pytest
python-dateutil
teuthology
```

Reply:
Nice improvement, I've updated the instructions and added the requirements.txt